aws-billing/aws-billing.json

```diff
@@ -1,5 +1,15 @@
 {
-  "__inputs": [],
+  "__inputs": [
+    {
```

Comment:
This should be deleted - otherwise import is asking for that input. Unfortunately, dashboard export thinks this is variable, which needs to defined, but it is not true. I always deleting this part + related parts

Reply:
yeah my bad, i thought that it would be a nice feature 😅 

Have removed it.